<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>518</width>
    <height>422</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabFuncionario">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>10</y>
      <width>1681</width>
      <height>411</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <property name="tabBarAutoHide">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Cadastrar Usuário</string>
     </attribute>
     <widget class="QTableWidget" name="tabelaUsuarios">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>471</width>
        <height>191</height>
       </rect>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>94</number>
      </attribute>
      <column>
       <property name="text">
        <string>Id</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Usuário</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Nome</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Função</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Matricula</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Usuario</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Senha</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Nome</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Função</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>100</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Matricula</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="inputUsuario">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>10</y>
        <width>371</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="inputSenha">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>40</y>
        <width>371</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="inputNome">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>70</y>
        <width>371</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="inputFuncao">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>100</y>
        <width>151</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="inputMatricula">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>100</y>
        <width>131</width>
        <height>25</height>
       </rect>
      </property>
      <property name="inputMask">
       <string>9999999999</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="maxLength">
       <number>10</number>
      </property>
     </widget>
     <widget class="QPushButton" name="btnAdicionar">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Adicionar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnExcluir">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>140</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Excluir</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnAlterar">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>140</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Alterar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnLogout">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>140</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Logout</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Funcionários</string>
     </attribute>
     <widget class="QTableWidget" name="tabelaFuncionario">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>100</y>
        <width>501</width>
        <height>281</height>
       </rect>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>125</number>
      </attribute>
      <column>
       <property name="text">
        <string>Id</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Nome</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Função</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Matricula</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Nome:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="inputNomeFuncionario">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>10</y>
        <width>421</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnBuscar">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>60</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Buscar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnExcluirFuncionario">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>60</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Excluir</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnImprimir">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>60</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Imprimir</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="produtos">
     <attribute name="title">
      <string>Produtos</string>
     </attribute>
     <widget class="QLineEdit" name="inputCodigo">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>10</y>
        <width>111</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableWidget" name="tabelaProduto">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>90</y>
        <width>501</width>
        <height>281</height>
       </rect>
      </property>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>125</number>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderMinimumSectionSize">
       <number>22</number>
      </attribute>
      <column>
       <property name="text">
        <string>Codigo</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Descricao</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="btnAdicionarProduto">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>50</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Adicionar</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnExcluirProduto">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>50</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Excluir</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Código</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="inputDescricao">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>10</y>
        <width>251</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>10</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Descrição</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
